public/app/features/alerting/unified/triage/Workbench.tsx

```diff
@@ -168,19 +177,37 @@ export function Workbench({ domain, data, queryRunner, groupBy }: WorkbenchProps
             domain={domain}
             queryRunner={queryRunner}
           >
-            <ScrollContainer height="100%" width="100%" scrollbarWidth="none" showScrollIndicators>
-              {isLoading ? (
+            <ScrollContainer height="100%" width="100%" scrollbarWidth="none" showScrollIndicators={showData}>
+              {isLoading && (
                 <>
                   <GenericRowSkeleton key="skeleton-1" width={leftColumnWidth} depth={0} />
                   <GenericRowSkeleton key="skeleton-2" width={leftColumnWidth} depth={0} />
                   <GenericRowSkeleton key="skeleton-3" width={leftColumnWidth} depth={0} />
                 </>
-              ) : (
+              )}
+              {showEmptyState && (
+                <div className={styles.emptyStateContainer}>
+                  <EmptyState
+                    variant="not-found"
+                    message={hasActiveFilters ? 'No matching instances found' : 'No firing or pending instances'}
+                  >
+                    {hasActiveFilters ? (
+                      <Trans i18nKey="alerting.triage.no-matching-instances-with-filters">
+                        No alert instances match your current set of filters for the selected time range.
+                      </Trans>
+                    ) : (
+                      <Trans i18nKey="alerting.triage.no-firing-or-pending-instances">
+                        You have no alert instances in a firing or pending state.
+                      </Trans>
+                    )}
+                  </EmptyState>
+                </div>
+              )}
+              {showData &&
                 dataSlice.map((row, index) => {
                   const rowKey = generateRowKey(row, index);
                   return renderWorkbenchRow(row, leftColumnWidth, domain, rowKey, enableFolderMeta);
-                })
-              )}
+                })}
               {hasMore && <LoadMoreHelper handleLoad={() => setPageIndex((prevIndex) => prevIndex + 1)} />}
             </ScrollContainer>
           </WorkbenchProvider>
```

Comment:
I wonder if we could simplify the code if we skipped rendering of the whole WorkbenchProvider and ScrollContainer components if there is no data?
Rendering the empty state inside them adds a bit of complexity